osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -957,10 +957,18 @@ private void carouselBeatmapsLoaded()
 
         private void updateVisibleBeatmapCount()
         {
-            // Intentionally not localised until we have proper support for this (see https://github.com/ppy/osu-framework/pull/4918
-            // but also in this case we want support for formatting a number within a string).
             int carouselCountDisplayed = Carousel.CountDisplayed;
-            FilterControl.InformationalText = carouselCountDisplayed != 1 ? $"{carouselCountDisplayed:#,0} matches" : $"{carouselCountDisplayed:#,0} match";
+
+            if (FilterControl.CurrentTextSearch.Value == string.Empty)
+            {
+                FilterControl.InformationalText = $"{carouselCountDisplayed} beatmaps available";
```

Comment:
This brings back the terminology conflict in https://github.com/ppy/osu/pull/23000#issuecomment-1510143838 (first run setup counting "beatmaps" and song select counting "beatmap difficulties", which was too long a label that it had to be changed to be ambiguous).

Can be `X available`, but that looks wrong.

Reply:
Ah I see, if anything shouldn't the first run import text be changed? Either changing the text to say "mapsets" or changing the number to match stable "beatmaps" would work. The disparity between the import number and the number I got in the stable dialogue is quite confusing irrespective of this change so a change there would be beneficial anyway. 
![image](https://github.com/user-attachments/assets/7f745fac-8356-47d2-866b-bddbcabcc38a)

Reply:
Based on https://osu.ppy.sh/wiki/en/Beatmap, stable wording is wrong. Also, the term "mapsets" are rarely used in the wiki.

I just want consistency in wiki, lazer/stable, and web.

Reply:
Should it not just read `X matching difficulties` if we want it to be more explicit? Also I don't get why it becomes "matches" when filtering.

Reply:
Yes, I'm fine with that and `X difficulties available` if what you believe in https://github.com/ppy/osu/pull/23000#issuecomment-1486291286 (with words just swapped) changed.

@jhk2601 this new string also doesn't account for plurality.

And search is not the only filter. This PR doesn't take account of the difficulty range, collections, ruleset, conversion, etc.

Reply:
hmm this is a great point, the "available difficulties" text in stable appeared on the main menu which avoided similar concerns. Might just close this and open a discussion since its less of an "easy fix" than I imagined, moving away from "matches" altogether somehow might be a better idea..